src/app/styles/global.scss

```diff
@@ -0,0 +1,112 @@
+@import 'app/styles/variables';
+
+:root {
+	--white: #{$white};
```

Comment:
ahh.. previous comment was a bit early. looks like we have both sass and native css vars - carry on :p

Reply:
as I wrote in the PR comments, we'll be keeping the old custom properties for compatibility purposes, but we'll eventually phase them out and redesign components that rely on the custom properties